src/coinjoin/coinjoin.cpp

```diff
@@ -4,24 +4,25 @@
 
 #include <coinjoin/coinjoin.h>
 
-#include <bls/bls.h>
 #include <chain.h>
 #include <chainparams.h>
 #include <consensus/validation.h>
-#include <governance/common.h>
-#include <instantsend/instantsend.h>
-#include <llmq/chainlocks.h>
-#include <masternode/node.h>
-#include <masternode/sync.h>
 #include <messagesigner.h>
 #include <netmessagemaker.h>
 #include <txmempool.h>
 #include <util/moneystr.h>
 #include <util/system.h>
 #include <util/translation.h>
 #include <validation.h>
-
 #include <tinyformat.h>
+
+#include <bls/bls.h>
+#include <chainlocks/chainlocks.h>
+#include <governance/common.h>
+#include <instantsend/instantsend.h>
+#include <masternode/node.h>
+#include <masternode/sync.h>
```

Comment:
nit: should not split it to own group IMO, it's not something special here about them

Reply:
IMO Dash-specific headers should be grouped separately, gives a quick glance of what Dash-specific dependencies are present. Would like this even in Dash-specific code as it better expresses interplay with various components, prefer to do grouping where it is trivial to do.

Reply:
Should keep them on top then maybe?

Reply:
I agree with your point about backports, but `src/coinjoin/coinjoin.cpp` is our file, there's no backports to this file.